task9/repository/roverClient.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+import { config } from "../config/config.js";
+
+export const getPhotos = async (apiKey) => {
+    const response = await axios.get(config.nasaRoverUrl, {
```

Comment:
u can use here block --> try (const response = ...) / catch (throw new Error ...)

Reply:
hmmm:) ok, why? but no problem, then let's not add him